.gitignore

```diff
@@ -10,3 +10,4 @@ __pycache__
 src/_features.h
 netplan/_features.py
 dbus/io.netplan.Netplan.service
+build/
```

Comment:
This is not necessary. The builddir can be named anything, so you cannot predict it in gitignore... but Meson does know at the time of generation, and will create a file named `builddir/.gitignore` (or, if you prefer `build/`, it will create `build/.gitignore`) with the contents `*`, in order to ignore the entire contents of the generated build tree.

Meson knows that no files in a generated build tree ever have a reason to be checked into git, so it is easy to do the right thing here. :)

Reply:
Thanks!